docs/api.md

```diff
@@ -101,15 +101,15 @@ Where the fields are as follows:
 ### Error codes
 
 - `RS_UNKNOWN` is a catch-all error when the appliation does not have a specific error.
-- `RS_METHOD_NOT_ALLOWED` is reported when you have used the wrong method for a service. E.g. GET instead of POST.
+- `RS_METHOD_NOT_ALLOWED` is reported when you have used the wrong method for an endpoint. E.g. GET instead of POST.
 - `RS_DISALLOWED_APP` is reported when a report was rejected due to the report being sent from an unsupported
    app (see the `allowed_app_names` config option).
 - `RS_BAD_HEADER` is reported when a header was not able to be parsed, such as `Content-Length`.
 - `RS_CONTENT_TOO_LARGE` is reported when the reported content size is too large.
 - `RS_BAD_CONTENT` is reported when the reports content could not be parsed.
-- `RS_REJECTED` is is reported when the submission could be understood but was rejected by `rejection_conditions`.
+- `RS_REJECTED` is reported when the submission could be understood but was rejected by `rejection_conditions`.
   This is the default value, see below for more information.
 
 In addition to these error codes, the configuration allows application developers to specify specific error codes
-for report rejection (see the `rejection_conditions` config option). Consult the administrator of your rageshake
-server in order to determine what error codes may be presented.
+for report rejection under the `RS_REJECTED_*` namespace. (see the `rejection_conditions` config option). Consult the
+administrator of your rageshake server in order to determine what error codes may be presented.
```

Comment:
ok, so what does `RS_` mean here, and do we really need it?

Reply:
RageShake, and it just felt like good form to use a namespace for the error codes. If you have strong feelings, I can remove it.

Reply:
We don't use that abbreviation elsewhere ("rageshake" is not long), and I can't really see the need for a prefix here. Could you remove it?

Reply:
Done :+1: